contents.json

```diff
@@ -5900,6 +5900,12 @@
       "description": "Mock Alamofire and URLSession requests without touching your code implementation",
       "homepage": "https://github.com/WeTransfer/Mocker",
       "tags": ["alamofire", "urlsession", "mock"]
+    }, {
+      "title": "ContainerController",
+      "category": "ui",
+      "description": "UI Component. This is a copy swipe-panel from app: Apple Maps, Stocks. Swift version",
```

Comment:
@mrustaa [The description usually should not include "written in Swift" or "in Swift" as everything on the list would have that](https://github.com/matteocrippa/awesome-swift/blob/master/.github/CONTRIBUTING.md). Could you remove the "Swift version" from the description?

Reply:
ok/ fixed